01.w_Defects/invalid_memory_access.c

```diff
@@ -472,8 +472,11 @@ int invalid_memory_access_014 (int flag)
 	}
 	if(ptr!=NULL)
 	{
+                // TODO: goto jumps past line with error so it does not actually execute
    		goto my_label2;
+my_label3:
         ret = ptr[2];/*Tool should detect this line as error*/ /*ERROR:Invalid memory access to already freed area*/
```

Comment:
this line wasn't being executed before so I rearranged the gotos so that it gets executed, but still is dependent on the control flow of the goto.